app/assets/images/payment_icons/swish.svg

```diff
@@ -1 +1 @@
Gradient id="pi-swish-SVGID_1_" cx="22.9" cy="8.5" r="12.988" gradientTransform="matrix(1 0 0 1.001 0 -.008)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fdd626"/><stop offset=".314" stop-color="#eb5253"/><stop offset=".506" stop-color="#eb4f65"/><stop offset=".639" stop-color="#e739cb"/><stop offset=".8" stop-color="#9e16ec"/><stop offset="1" stop-color="#5702f0"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_1_)" d="M22.25 7.95c.566.333 1.1.767 1.6 1.3 1.233 1.233 1.983 2.7 2.25 4.4.034.233.051.449.051.649.033.2.05.384.05.55 0 .267-.017.551-.05.851-.4.7-.9 1.35-1.5 1.95C23.083 19.217 21.2 20 19 20c-1.1 0-2.133-.2-3.1-.6.667.233 1.367.35 2.1.35 1.866 0 3.434-.634 4.7-1.9.434-.466.8-.949 1.1-1.449.033-.067.066-.134.101-.2.533-.934.8-1.95.8-3.05 0-1.867-.667-3.45-2-4.75l-.451-.451z"/><radialGradient id="pi-swish-SVGID_2_" cx="23.45" cy="7.2" r="8.925" gradientTransform="matrix(1 0 0 1.0014 0 -.01)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fdd626"/><stop offset=".365" stop-color="#f17345"/><stop offset=".663" stop-color="#f00912"/><stop offset=".878" stop-color="#ee0920"/><stop offset="1" stop-color="#e30767"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_2_)" d="M26.15 15.7c.033-.3.05-.584.05-.851 0-.166-.017-.35-.05-.55 0-.2-.017-.416-.051-.649-.267-1.7-1.017-3.167-2.25-4.4a7.206 7.206 0 00-1.6-1.3L24 5.8c.2.167.417.35.65.55a7.173 7.173 0 011.5 2c.567 1.1.85 2.317.85 3.65 0 .4-.017.8-.05 1.2-.167.866-.433 1.7-.8 2.5z"/><radialGradient id="pi-swish-SVGID_3_" cx="14.8" cy="16.8" r="9.25" gradientTransform="matrix(1 0 0 1.0014 0 -.023)" gradientUnits="userSpaceOnUse"><stop offset=".086" stop-color="#424ee9"/><stop offset=".447" stop-color="#58c9be"/><stop offset=".71" stop-color="#59cab5"/><stop offset="1" stop-color="#8ce785"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_3_)" d="M12.1 7.95c-.067.433-.1.867-.1 1.3v.6c0 .1.017.217.05.35.167 1.767.934 3.333 2.3 4.7.433.466.9.85 1.4 1.149l-1.8 2.2c-.2-.167-.4-.366-.6-.6a7.73 7.73 0 01-1.3-1.7C11.35 14.783 11 13.467 11 12c0-.233.017-.483.05-.75.1-1.2.45-2.3 1.05-3.3z"/><radialGradient id="pi-swish-SVGID_4_" cx="14.65" cy="14.95" r="12.813" gradientTransform="matrix(1 0 0 1.001 0 -.015)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#424ee9"/><stop offset=".392" stop-color="#58c9be"/><stop offset=".765" stop-color="#6cd82e"/><stop offset="1" stop-color="#e5db2a"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_4_)" d="M15.75 16.05c-.5-.3-.967-.684-1.4-1.149-1.367-1.367-2.133-2.934-2.3-4.7A1.499 1.499 0 0112 9.85v-.6c0-.433.033-.867.1-1.3a7.98 7.98 0 011.25-1.6C14.917 4.783 16.8 4 19 4c1.233 0 2.35.233 3.35.7-.7-.233-1.484-.35-2.35-.35-1.8 0-3.367.633-4.7 1.9a5.93 5.93 0 00-1.05 1.4c-.067.133-.15.25-.25.35-.4.9-.6 1.883-.6 2.95 0 1.833.633 3.399 1.9 4.7.133.133.283.267.45.4z"/></g></svg>
```

Comment:
Same comments applied here: unused id attributes and border needs to follow other icons.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-swish"><title id="pi-swish">Swish</title><g fill-rule="evenodd" clip-rule="evenodd"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><g><radialGradient id="pi-swish-SVGID_1_" cx="22.9" cy="8.5" r="12.988" gradientTransform="matrix(1 0 0 1.001 0 -.008)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fdd626"/><stop offset=".314" stop-color="#eb5253"/><stop offset=".506" stop-color="#eb4f65"/><stop offset=".639" stop-color="#e739cb"/><stop offset=".8" stop-color="#9e16ec"/><stop offset="1" stop-color="#5702f0"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_1_)" d="M22.25 7.95c.566.333 1.1.767 1.6 1.3 1.233 1.233 1.983 2.7 2.25 4.4.034.233.051.449.051.649.033.2.05.384.05.55 0 .267-.017.551-.05.851-.4.7-.9 1.35-1.5 1.95C23.083 19.217 21.2 20 19 20c-1.1 0-2.133-.2-3.1-.6.667.233 1.367.35 2.1.35 1.866 0 3.434-.634 4.7-1.9.434-.466.8-.949 1.1-1.449.033-.067.066-.134.101-.2.533-.934.8-1.95.8-3.05 0-1.867-.667-3.45-2-4.75l-.451-.451z"/><radialGradient id="pi-swish-SVGID_2_" cx="23.45" cy="7.2" r="8.925" gradientTransform="matrix(1 0 0 1.0014 0 -.01)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fdd626"/><stop offset=".365" stop-color="#f17345"/><stop offset=".663" stop-color="#f00912"/><stop offset=".878" stop-color="#ee0920"/><stop offset="1" stop-color="#e30767"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_2_)" d="M26.15 15.7c.033-.3.05-.584.05-.851 0-.166-.017-.35-.05-.55 0-.2-.017-.416-.051-.649-.267-1.7-1.017-3.167-2.25-4.4a7.206 7.206 0 00-1.6-1.3L24 5.8c.2.167.417.35.65.55a7.173 7.173 0 011.5 2c.567 1.1.85 2.317.85 3.65 0 .4-.017.8-.05 1.2-.167.866-.433 1.7-.8 2.5z"/><radialGradient id="pi-swish-SVGID_3_" cx="14.8" cy="16.8" r="9.25" gradientTransform="matrix(1 0 0 1.0014 0 -.023)" gradientUnits="userSpaceOnUse"><stop offset=".086" stop-color="#424ee9"/><stop offset=".447" stop-color="#58c9be"/><stop offset=".71" stop-color="#59cab5"/><stop offset="1" stop-color="#8ce785"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_3_)" d="M12.1 7.95c-.067.433-.1.867-.1 1.3v.6c0 .1.017.217.05.35.167 1.767.934 3.333 2.3 4.7.433.466.9.85 1.4 1.149l-1.8 2.2c-.2-.167-.4-.366-.6-.6a7.73 7.73 0 01-1.3-1.7C11.35 14.783 11 13.467 11 12c0-.233.017-.483.05-.75.1-1.2.45-2.3 1.05-3.3z"/><radialGradient id="pi-swish-SVGID_4_" cx="14.65" cy="14.95" r="12.813" gradientTransform="matrix(1 0 0 1.001 0 -.015)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#424ee9"/><stop offset=".392" stop-color="#58c9be"/><stop offset=".765" stop-color="#6cd82e"/><stop offset="1" stop-color="#e5db2a"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#pi-swish-SVGID_4_)" d="M15.75 16.05c-.5-.3-.967-.684-1.4-1.149-1.367-1.367-2.133-2.934-2.3-4.7A1.499 1.499 0 0112 9.85v-.6c0-.433.033-.867.1-1.3a7.98 7.98 0 011.25-1.6C14.917 4.783 16.8 4 19 4c1.233 0 2.35.233 3.35.7-.7-.233-1.484-.35-2.35-.35-1.8 0-3.367.633-4.7 1.9a5.93 5.93 0 00-1.05 1.4c-.067.133-.15.25-.25.35-.4.9-.6 1.883-.6 2.95 0 1.833.633 3.399 1.9 4.7.133.133.283.267.45.4z"/></g></svg>
```